src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectActivityScore.java

```diff
@@ -26,45 +29,76 @@
 public class ProjectActivityScore extends FeatureBasedScore {
 
   /**
-   * y(n) = 7.1 / (1 + e ^ (0.04 * (50 - n))),
+   * The maximum sub-score based on a number of commits in the last three month.
+   */
+  private static final double MAX_COMMITS_SUB_SCORE = 5.1;
+
+  /**
+   * A logistic function which is used to convert a number of commits to a sub-score:
+   * <pre>y(n) = MAX_COMMITS_SUB_SCORE / (1 + e ^ (0.02 * (200 - n)))</pre>,
    * where n is a number of commits in the last three months.
-   * y(n) belongs to the interval (0.8, 7.1) when n > 0
-   * Number of commits can contribute up to 7 to the overall project activity score.
+   * The function belongs to the interval [0.0, MAX_COMMITS_SUB_SCORE) when n >= 0.
    */
-  static final Logistic LOGISTIC_FOR_NUMBER_OF_COMMITS
-      = new Logistic(7.1, 50, 0.04, 1, 0, 1);
+  private static final Logistic LOGISTIC_FOR_NUMBER_OF_COMMITS
+      = logistic(MAX_COMMITS_SUB_SCORE, 0.02, 200);
```

Comment:
Any specific reason these number 0.02 and 200 were used?

Reply:
These are parameters of the logistic function. First, I adjusted the test vectors. Then, I manually adjusted these parameters to satisfy the test vectors. Roughly, `200` means that 400 commits produces almost the max value of the function (400 = 200 * 2, and the logistic function is symmetric). `0.02` means that the function doesn't grow too fast (in other words, it should be closer to the linear function rather that to the step-function). If you plot this function, you can see how actually a number of commits affects the score.

Reply:
Ok understood.